### PR TITLE
Blog post What's new in ZuulV3

### PR DESCRIPTION
**Pull request by TristanCacqueray**

### CODE REVIEW

File: source/blog/2017-07-31-whats-new-in-zuulV3.html.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+This configuration describes a *downstream* tenant with two sources. Gerrit
+is a local gerrit instance and openstack.org is the review.openstack.org
+service. For each sources, there are 2 types of projects:
+
+* config-projects hold configuration information such as logserver access.
+  Jobs defined in config-projects run with elevated privileges.
+* untrusted-projects are projects being tested or deployed.
+
+The openstack-infra/zuul-jobs has special settings discussed bellow.
```
Comment (morucci) [inline source/blog/2017-07-31-whats-new-in-zuulV3.html.md]: s/bellow/below/

File: source/blog/2017-07-31-whats-new-in-zuulV3.html.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+This is great news for Third Party CIs that will easily be able to re-use
+upstream jobs such as tox-docs or tox-py35 with their convenient
+post-processing of unittest results.
+
+
+## In-repository configuration
+
+The new distributed configuration enables a more streamlined workflow.
+Indeed, the pipelines and projects are now defined in the project's repository
```
Comment (morucci) [inline source/blog/2017-07-31-whats-new-in-zuulV3.html.md]: Indeed, pipelines and ... (not sure you need the 'the' here)

File: source/blog/2017-07-31-whats-new-in-zuulV3.html.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+which allows changes to be tested before merging.
+
+Traditionaly, projects' CI needed to be configured in two steps: first, the jobs
+were defined, then a test change was rechecked until the configuration was correct.
```
Comment (morucci) [inline source/blog/2017-07-31-whats-new-in-zuulV3.html.md]: until the job was correct ?

File: source/blog/2017-07-31-whats-new-in-zuulV3.html.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+```yaml
+- project:
+    name: restfuzz
+    check:
+      jobs:
+        - tox-py35
+```
+
+Zuul will look for a zuul.yaml file or a zuul.d directory as well as hidden
+versions prefixed by a '.'. The project can also define it's own jobs.
```
Comment (morucci) [inline source/blog/2017-07-31-whats-new-in-zuulV3.html.md]: s/it's/its/

**Comment by TristanCacqueray** [comment]
Thanks morucci, next update should correct all the typos you spotted!

**Pull request merged**